### ISSUE TITLE
Continue a training with load_model = True

### ISSUE THREAD

**Issue by Fangh** [Unity-Technologies/ml-agents#205]
Hi.
I have 7 questions about continue a training session.

I make an AI learn within 500k steps with my hyperparameter _max_steps_ = **5e5**.
Then I change the hyperparameter _load_model_ to **True** and _max_steps_ to **1e6**.

I hoped my AI continue to learn and with 500k steps it was good but I wanted it to be better.

These are the result in tensorboard :

![2017-12-23 23_32_08-tensorboard](https://user-images.githubusercontent.com/6020679/34323087-092fc000-e83a-11e7-9db0-d6fa74efc452.png)

The Entropy continue to decrease after 500k. Which in what I expected. The AI tend to "understand" what it is doing.

![2017-12-23 23_32_21-tensorboard](https://user-images.githubusercontent.com/6020679/34323090-41c909f8-e83a-11e7-89e6-1c0cb0d9097c.png)

1) Why does the Learning rate decrease to zero when it does its first session of learning ?
2) Why does the Learning rate goes to its starting point when starting a new session of learning ?
3) Why the Learning rate doesn't follow the same "motion" the second time ?

![2017-12-23 23_31_58-tensorboard](https://user-images.githubusercontent.com/6020679/34323102-b43acd64-e83a-11e7-8df6-b416f6e1dd2a.png)

This is what bother me the most...
4) How can the cumulative rewards decrease after doing the same thing 800k times ?
5) I was playing video games while the whole second training session. Does using my CPU affect the learning process ?

![2017-12-23 23_32_26-tensorboard](https://user-images.githubusercontent.com/6020679/34323122-3a428b22-e83b-11e7-9ace-7b94523f2748.png)

6) Why does the policy loss is going crazy ? It should stabilize around 0 if I understood correctly.

![2017-12-23 23_32_35-tensorboard](https://user-images.githubusercontent.com/6020679/34323123-58ed4c74-e83b-11e7-9ba5-48e6b63d5d98.png)

7) I never understood theses graph. Do they have any value to understand what is going on ?

Thank you for your answer. If you can't answer to all of them, please answer those you know. Thank you.

**Comment by tgzeror** [comment]
Same problem, load_model does not have any  effect 
I also want to continue a training session

**Comment by menondj** [comment]
I think the max_steps in the second run should be 5e5+1e6

**Comment by vladimiroster** [comment]
Hi @Fangh, thanks for reaching out to us. Hopefully you were able to resolve your issue.  We are closing this due to inactivity, but if you need additional assistance, feel free to reopen the issue.
